src/pages/settings/Security/TwoFactorAuth/Steps/VerifyStep.js

```diff
@@ -0,0 +1,175 @@
+import React, {useEffect} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {ScrollView, View} from 'react-native';
+import PropTypes from 'prop-types';
+import withLocalize, {withLocalizePropTypes} from '../../../../../components/withLocalize';
+import compose from '../../../../../libs/compose';
+import * as Session from '../../../../../libs/actions/Session';
+import styles from '../../../../../styles/styles';
+import Button from '../../../../../components/Button';
+import Text from '../../../../../components/Text';
+import ONYXKEYS from '../../../../../ONYXKEYS';
+import TextLink from '../../../../../components/TextLink';
+import Clipboard from '../../../../../libs/Clipboard';
+import FixedFooter from '../../../../../components/FixedFooter';
+import * as Expensicons from '../../../../../components/Icon/Expensicons';
+import PressableWithDelayToggle from '../../../../../components/Pressable/PressableWithDelayToggle';
+import TwoFactorAuthForm from '../TwoFactorAuthForm';
+import QRCode from '../../../../../components/QRCode';
+import expensifyLogo from '../../../../../../assets/images/expensify-logo-round-transparent.png';
+import CONST from '../../../../../CONST';
+import StepWrapper from "../StepWrapper/StepWrapper";
+import useTwoFactorAuthContext from "../TwoFactorAuthContext/useTwoFactorAuth";
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    account: PropTypes.shape({
+        /** Whether this account has 2FA enabled or not */
+        requiresTwoFactorAuth: PropTypes.bool,
+
+        /** Secret key to enable 2FA within the authenticator app */
+        twoFactorAuthSecretKey: PropTypes.string,
+
+        /** User primary login to attach to the authenticator QRCode */
+        primaryLogin: PropTypes.string,
+
+        /** User is submitting the authentication code */
+        isLoading: PropTypes.bool,
+
+        /** Server-side errors in the submitted authentication code */
+        errors: PropTypes.objectOf(PropTypes.string),
+    }),
+};
+
+const defaultProps = {
```

Comment:
Use prop destructuring with default values instead of `defaultProps` -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring